src/otx/recipe/detection/atss_mobilenetv2.yaml

```diff
@@ -79,6 +81,7 @@ overrides:
                 - 736
               keep_ratio: false
               transform_bbox: false
```

Comment:
It's just a question. Doesn't bbox have to be changed during validation and test?

Reply:
For metric calculation, original bbox is used, so we don't need to transform it like mmx.